test/functions.jl

```diff
@@ -204,4 +205,149 @@
             @test g.affine_terms == MOI.VectorAffineTerm.([1, 2, 1], MOI.ScalarAffineTerm.([3, 1, 1], [x, x, y]))
         end
     end
+    @testset "Conversion to canonical form" begin
+        function _isapprox(f1::T, f2::T) where {T <: Union{MOI.ScalarAffineFunction, MOI.VectorAffineFunction}}
+            ((MOIU.termindices.(f1.terms) == MOIU.termindices.(f2.terms)) &&
+             (MOIU._constant(f1) ≈ MOIU._constant(f2)) &&
+             (MOIU.coefficient.(f1.terms) ≈ MOIU.coefficient.(f2.terms)))
+        end
+        function test_canonicalization(f::T, expected::T) where {T <: Union{MOI.ScalarAffineFunction, MOI.VectorAffineFunction}}
+            @test MOI.Utilities.iscanonical(expected)
```

Comment:
Inconsistent use of `MOIU` and `MOI.Utilities`.

Reply:
fixed